fr/releases/centreon-core.md

```diff
@@ -18,6 +18,66 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web release notes
 
+### 20.04.6
+
+> Newly shared views do not break widget preferences.
+> But, if you have already broken widget preferences for users who add a shared
+> view, you'll need to:
+>
+> * Login centreon web with the user who shares the custom view
+> * Switch to the custom view with broken preferences for other users
+> * Click on "Share view", and then click on "Share"
+>
+> This will restore preferences for other users
+
+#### Enhancements
+
+* [Event View] Be able to create & save filters
```

Comment:
to be consistent with line 39
```suggestion
* [Event View] Be able to create and save filters
```

Reply:
some new fixes have been added in the version and not tested yet, including one new today so I will check the list when available

Reply:
this RN must be cherry-picked on branch 20.04.x of the documentation for it to be released in the 20.04 documentation